velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -140,14 +140,57 @@ void toDuckDbFilter(
       }
       break;
     }
+    case common::FilterKind::kBigintValuesUsingHashTable: {
+      auto valuesFilter =
+          static_cast<common::BigintValuesUsingHashTable*>(filter);
+      const auto& values = valuesFilter->values();
+
+      if (values.size() == 1) {
+        filters.PushFilter(colIdx, constantEqualFilter(*values.begin()));
+      } else {
+        std::vector<::duckdb::Value> valuesList;
+        valuesList.reserve(values.size());
+        for (const auto& value : values) {
+          valuesList.push_back(makeValue(type, value));
+        }
+
+        filters.PushFilter(
+            colIdx,
+            constantFilter(
+                ::duckdb::ExpressionType::COMPARE_IN,
```

Comment:
I'm hitting DASSERT(0) in ApplyFilter for COMPARE_IN. Looks like it is not supported. @Mytherin Mark, how hard would it be to add support for COMPARE_IN filters to Parquet reader? CC: @majetideepak

Reply:
Likely not so difficult for primitive types. Currently we transform `IN` filters into a chain of `OR` filters for smaller lists, or transform it into a join and don't push down at all if the `IN` list is large. Happy to review a PR!

Reply:
> Currently we transform IN filters into a chain of OR filters for smaller lists, or transform it into a join and don't push down at all if the IN list is large.

@Mytherin Thank you for explaining, Mark.